packages/autotools-scripts/src/cli.ts

```diff
@@ -48,6 +48,12 @@ program
 .option('-f, --files [pattern]', 'metadata file pattern')
 .action((options) => {
   const entry = glob.sync(path.join(projectPath, options.files ? options.files : defaultMetaGlob));
+
+  if (!process.env.EYES_API_KEY) {
```

Comment:
I meant this should be in the `eyesTest` function. Another option is to pass `eyesApiKey` as a param to `eyesTest()`, and then it would be validated inside of that function.

Also if we're going to do `require('dotenv').config();` we should do it in this file.